packages/core/src/api.ts

```diff
@@ -256,6 +257,7 @@ export const fetch: FetchFunc = async (
   progressEmitter?,
   accounts?,
   ignoreStateElemIdMapping?,
+  ChangeDetection?,
```

Comment:
args should start with lowercase
```suggestion
  changeDetection?,
```

Reply:
good catch, should have been `withChangeDetection`